src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -861,7 +870,8 @@ public void execute() throws GitException, InterruptedException {
                 }
 
                 if (refspecs == null) {
-                    refspecs = Collections.singletonList(new RefSpec("+refs/heads/*:refs/remotes/" + origin + "/*"));
+                    refspecs = Collections.singletonList(
+                            new RefSpec("+refs/heads/*:refs/remotes/" + origin + "/*".trim()));
```

Comment:
As this hard-coded we can remove `trim()`. I was not sure if I should.

Reply:
The existing `trim` should be removed.  It won't change the RefSpec definition in any way because `"/*".trim()` is the same as `"/*"`